sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.0.0 (2022-09-29)
```

Comment:
Why did we revert the change in changelog?

Reply:
I copy this file from an old branch, reverted it.